pkg/razorpay/orders_test.go

```diff
@@ -0,0 +1,376 @@
+package razorpay
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/razorpay/razorpay-go"
+	"github.com/razorpay/razorpay-go/constants"
+
+	"github.com/razorpay/razorpay-mcp-server/pkg/razorpay/mocks"
+)
+
+func Test_CreateOrder(t *testing.T) {
+	createOrderPath := fmt.Sprintf(
+		"/%s%s",
+		constants.VERSION_V1,
+		constants.ORDER_URL,
+	)
+
+	tests := []struct {
+		name           string
+		mockHttpClient func() (*http.Client, *httptest.Server)
+		requestArgs    map[string]interface{}
```

Comment:
better to keep requestArgs first in all then mockHttpClient which has response after that - will make more readable and otehrs would also follow this - so please make that change also across.